cpp/src/community/legacy/egonet.cu

```diff
@@ -152,7 +150,7 @@ extract(raft::handle_t const& handle,
   // Construct the neighboors list concurrently
   for (vertex_t i = 0; i < n_subgraphs; i++) {
     auto worker_stream_view = handle.get_internal_stream_view(i);
```

Comment:
Don't need to create `worker_stream_view` here...

Reply:
Thanks for spotting this. However, it seems like the `worker_stream_view` object is used afterward and allows concurrent work in loops. I think a good fix would be to revert back `handle.get_thrust_policy()` to `rmm::exec_policy(worker_stream_view)` everywhere in the file. What do you think?

Reply:
I just pushed this solution. Please tell me if you prefer it to be done some other way.

Reply:
This looks better.  Thanks!